### PR TITLE
Fix a bunch of broken `https://github.com/DefinitelyTyped` owners

### PR DESCRIPTION
**Pull request by elibarzilay**

### CODE REVIEW

File: types/chai-like/index.d.ts [hunk 1]
```diff
@@ -1,6 +1,6 @@
 // Type definitions for chai-like 1.1
 // Project: https://github.com/zation/chai-like
-// Definitions by: Chayim Refael Friedman <https://github.com/DefinitelyTyped>
+// Definitions by: Chayim Refael Friedman <https://github.com/DefinitelyTyped/ChayimFriedman2>
```
Comment (orta) [inline types/chai-like/index.d.ts:3]:
```suggestion
// Definitions by: Chayim Refael Friedman <https://github.com/ChayimFriedman2>
```

File: types/field/index.d.ts [hunk 2]
```diff
@@ -1,8 +1,7 @@
 // Type definitions for field 1.0.1
 // Project: https://www.npmjs.com/package/field
-// Definitions by: Leo Liang <https://github.com/aleung/DefinitelyTyped>
+// Definitions by: Leo Liang <https://github.com/DefinitelyTyped/aleung>
```
Comment (orta) [inline types/field/index.d.ts:3]:
```suggestion
// Definitions by: Leo Liang <https://github.com/aleung>
```

File: types/vhost/index.d.ts [hunk 3]
```diff
@@ -1,10 +1,13 @@
...
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
 import express = require('express');
 
 declare function vhost(hostname: string | RegExp, handler: express.Handler): express.Handler;
 
 export = vhost;
+> a1097f67a2 Paul Maréchal (7 months ago) 🤖 Merge PR #49520 vhost: fix return type by @marechal-p
+> 46cb9e4438 Vincenzo Chianese (1 year, 3 months ago) Update index.d.ts (#43561)
+> e21346be33 Vincenzo Chianese (1 year, 3 months ago) feat: add vhost types (#43467)
```
Comment (orta) [inline types/vhost/index.d.ts:13]: ^

Comment (orta) [inline types/vhost/index.d.ts:13]:
```suggestion

```

**Comment by elibarzilay** [comment]
Thanks -- all fixed...

**Comment by Ailrun** [comment]
I don't want to manage the type definition again. Please remove me from the file.

**Review by orta — APPROVED** [review]

**Review by Kocal — APPROVED** [review]

**Comment by Kocal** [comment]
Thanks!

**Comment by elibarzilay** [comment]
@Ailrun, this is only sorting out the bogus fields (which in some cases required going over ancient commits).  Once I merge this, I'll do another PR to remove you...  (And you'll probably see those pings too.)

**Comment by Ailrun** [comment]
@elibarzilay The problem is I'm the only manager of the definition afaik. It's not possible to remove the last manager from the file, and that's why somebody (I forget whom) suggested to use DefinitelyTyped instead of specific author.

**Comment by elibarzilay** [comment]
@Ailrun: sorry, I missed your comment and needed to merge this for another fix.

Anyway, yes, I'd change it back to DT as an owner, with a name of "nobody" -- but I only added you to `figlet/index.d.ts`, do you want to remove just that one or from the other two (`color-name`, `gradient-string`) as well?

**Comment by Ailrun** [comment]
@elibarzilay Yes, it would be great.

**Pull request merged**